react/src/views/Box/components/BoxDetails.tsx

```diff
@@ -1,312 +1,306 @@
 
 interface BoxDetailsProps {
-  boxData:
-    | BoxByLabelIdentifierQuery["box"]
-    | UpdateLocationOfBoxMutation["updateBox"];
-  onMoveToLocationClick: (locationId: string) => void;
-  onPlusOpen: () => void;
-  onMinusOpen: () => void;
-  onScrap: () => void;
-  onLost: () => void;
-  onAssignBoxToDistributionEventClick: (distributionEventId: string) => void;
-  onUnassignBoxFromDistributionEventClick: (
-    distributionEventId: string
-  ) => void;
-
+    boxData: BoxByLabelIdentifierQuery["box"] | UpdateLocationOfBoxMutation["updateBox"];
+    onMoveToLocationClick: (locationId: string) => void;
+    onPlusOpen: () => void;
+    onMinusOpen: () => void;
+    onScrap: () => void;
+    onLost: () => void;
+    onAssignBoxToDistributionEventClick: (distributionEventId: string) => void;
+    onUnassignBoxFromDistributionEventClick: (distributionEventId: string) => void;
 }
 
 const BoxDetails = ({
-  boxData,
-  onMoveToLocationClick,
-  onAssignBoxToDistributionEventClick,
-  onUnassignBoxFromDistributionEventClick,
-  onPlusOpen,
-  onMinusOpen,
-  onScrap,
-  onLost,
+    boxData,
+    onMoveToLocationClick,
+    onAssignBoxToDistributionEventClick,
+    onUnassignBoxFromDistributionEventClick,
+    onPlusOpen,
+    onMinusOpen,
+    onScrap,
+    onLost,
 }: BoxDetailsProps) => {
-  const statusColor = (value) => {
-    let color;
-    if (value === "Lost" || value === "Scrap") {
-      color = "#EB404A";
-    } else {
-      color = "#0CA789";
-    }
-    return color;
-  };
+    const statusColor = (value) => {
+        let color;
+        if (value === "Lost" || value === "Scrap") {
+            color = "#EB404A";
+        } else {
+            color = "#0CA789";
+        }
+        return color;
+    };
 
-  const { getDistroEventDetailUrlById } = useGetUrlForResourceHelpers();
+    const { getDistroEventDetailUrlById } = useGetUrlForResourceHelpers();
 
-  if (boxData == null) {
-    console.error("BoxDetails Component: boxData is null");
-    return <Box>No data found for a box with this id</Box>;
-  }
-
-  return (
-    <Flex
-      direction={["column", "column", "row"]}
-      alignItems={["center", "center", "flex-start"]}
-      w="100%"
-      justifyContent="center"
-    >
-      <Box
-        w={["100%", "80%", "30%", "30%"]}
-        border="2px"
-        mb={6}
-        pb={2}
-        backgroundColor="#F4E5A0"
-        mr={["0", "0", "4rem", "4rem"]}
-      >
-        <Flex pt={2} px={4} direction="row" justifyContent="space-between">
-          <Flex direction="column" mb={2}>
-            <Heading fontWeight={"bold"} as="h2">
-              Box {boxData.labelIdentifier}
-            </Heading>
-            <Flex>
-              <Text>
-                <b>State:&nbsp;</b>
-              </Text>
-              <Text color={statusColor(boxData.state)}><b>{boxData.state}</b></Text>
-            </Flex>
-          </Flex>
+    if (boxData == null) {
+        console.error("BoxDetails Component: boxData is null");
```

Comment:
maybe add a comment: "TODO: change error handling"

Reply:
@HaGuesto I think for this we also have task but that totally make sense to put a note